docs/passphrases.rst

```diff
@@ -21,21 +21,23 @@ Admin
 
 The admin will be using the *Admin Workstation* with Tails to connect to
 the *Application Server* and the *Monitor Server* using Tor and SSH. The tasks
-performed by the admin will require the following set of passphrases:
+performed by the admin will require the following set of credentials and 
+passphrases:
 
 -  A passphrase for the persistent volume on the Admin Live USB.
--  A master passphrase for the KeePassX password manager, which unlocks
-   passphrases to:
+-  A database maintained with Tails' KeePassX password manager, which is  
+   populated during the installation process and contains:
```

Comment:
For simplicity and flow, I would recommend saying something like:

"Additional credentials, which we recommend adding to Tails' KeePassX password manager during the installation:"